src/Actions/Ticket/CreateTicket.php

```diff
@@ -64,9 +67,35 @@ public function performAction(): Ticket
         $ticket->save();
 
         if (is_array($users)) {
-            $ticket->users()->sync($users);
+            foreach (data_get($ticket->users()->sync($users), 'attached', []) as $user) {
```

Comment:
if authenticatable user is the same as $user, then 2 exact same EventSubscriptions are being created.